### DISCUSSION TITLE
How to get the URL of the current tab?

### DISCUSSION THREAD

**Discussion by K0-RR** [brython-dev/brython#2333] [Q&A]
I'm trying to make a Chrome extension using Brython instead of JS. Is it possible to get the address of the page that is open?

**Comment by denis-migdal** [comment]
I am not sure if Brython as a Python-like API for that, but you can use the JS/DOM API:
1. Import window from browser.
2. then you have the current page with `window.location`.

**Reply by lulunac27a** [reply]
Use [`window.location.href`](https://developer.mozilla.org/en-US/docs/Web/API/Location/href) to get the current URL address, just like JavaScript does.
